core/src/main/java/org/elasticsearch/tribe/TribeService.java

```diff
@@ -104,6 +106,8 @@
             RestStatus.BAD_REQUEST, EnumSet.of(ClusterBlockLevel.METADATA_READ, ClusterBlockLevel.METADATA_WRITE));
     public static final ClusterBlock TRIBE_WRITE_BLOCK = new ClusterBlock(11, "tribe node, write not allowed", false, false,
             RestStatus.BAD_REQUEST, EnumSet.of(ClusterBlockLevel.WRITE));
+    private static final DeprecationLogger DEPRECATION_LOGGER =
+        new DeprecationLogger(Loggers.getLogger(TribeService.class));
```

Comment:
Also, I don't see why this needs to be a static field since it's used exactly once.